src/libraries/System.Net.Http/tests/PerformanceTests/HPackHuffmanBenchmark/HPackHuffmanBenchmark.csproj

```
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <Nullable>enable</Nullable>
```

```suggestion
    <Nullable>enable</Nullable>    
    <LangVersion>9.0</LangVersion>
```
Known issue for BDN 12.1, see: dotnet/BenchmarkDotNet#1452

---

Done